postgres-exporter/include/healthcheck

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -xe
+PORT=9187
+if ! [[ -z "$1" ]]; then
+  PORT=$1
+fi
+# This container is considered unhealthy when it can't
+# connect to the database.
+wget -O - http://localhost:$PORT/metrics | grep 'pg_up 1'
```

Comment:
Jim and I discussed this one, and decided that the metrics server for PG is better to remain running when PG is down. We need to test that if this pod is started without PG running, then PG starts, that the logs indicate that it established a connection (i.e. this program handles PG going down, then coming back up)